servers/propus/default.nix

```diff
@@ -0,0 +1,29 @@
+{ config, pkgs, inputs, lib, ... }: {
+  imports = [
+    inputs.serokell-nix.nixosModules.ec2
+  ];
+
+  networking.hostName = "propus";
+
+  services.jitsi-meet = {
```

Comment:
iirc there's some config options we should probably add to make it less shitty (like max framerate)